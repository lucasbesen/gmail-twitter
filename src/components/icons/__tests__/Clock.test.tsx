import React from 'react';
import renderer from 'react-test-renderer';

import { Clock } from '../';

it('should render Clock icon without error', () => {
  const component = renderer.create(<Clock />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
