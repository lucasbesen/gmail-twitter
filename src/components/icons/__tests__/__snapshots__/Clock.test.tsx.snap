// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Clock icon without error 1`] = `
<svg
  aria-hidden="true"
  className="sc-htoDjs eYbtof StyledIconBase-sc-bdy9j4 jbPeIR"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 16 16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M8 0a8 8 0 100 16A8 8 0 008 0zm2.293 11.707L7 8.414V4h2v3.586l2.707 2.707-1.414 1.414z"
  />
</svg>
`;
